app/mattermost.js

```diff
@@ -207,7 +209,11 @@ const handleOrientationChange = (orientation) => {
     }, 100);
 };
 
-export const handleManagedConfig = async (serverConfig) => {
+export const handleManagedConfig = async (eventFromEmmServer) => {
```

Comment:
Could you default eventFromEmmServer = false in the function parameter.  That way it's clear that function calls are not always passing an argument

Reply:
So if I understand this correctly. `eventFromEmmServer` is only used to avoid authentication if the managedConfig changed from the server.
There was never a real need for the `serverConfig`

Reply:
yes